### PR TITLE
viewpoints in admin_level

### PR DESCRIPTION
**Pull request by travisgrigsby**
This includes:
* Updates to the style file to push admin_level:XX from the rel down to the ways
* Support for turning admin_level:XX=Y into kind:XX=boundary_kind(Y)
* Tests to verify function

### CODE REVIEW

**Comment by nvkelso** [comment]
To enable easier dashed lines:

From: 

>       kind: mz_internal_dispute_mask
>       disputed_by: {col: disputed_by}

To: 

>       kind: mz_internal_dispute_mask
>       disputed: True
>       disputed_by: {col: disputed_by}

File: integration-test/2022-disputed-borders.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+                'admin_level:VN': '2.5',
+                'boundary': 'claim',
+                'name': 'Viewpoints on Disputed Administrative Boundaries',
+                'ne:brk_a3': 'B91',
+                'type': 'linestring',
+                'source': 'openstreetmap.org',
+            }),
+        )
+
+        with self.layers_in_tile(z, x, y) as layers:
```
Comment (nvkelso) [inline integration-test/2022-disputed-borders.py:53]: Please remove debug print statements before merging.

File: integration-test/2022-disputed-borders.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+            for layer in layers:
+                with self.features_in_tile_layer(z, x, y, layer) as features:
+                    for feature in features:
+                        print(feature)
+
+        self.assert_has_feature(
+            z, x, y, 'boundaries', {
+                'id': 726514231,
+                'kind:PS': 'locality',
```
Comment (nvkelso) [inline integration-test/2022-disputed-borders.py:62]: The Tilezen convention is all lowercase property names, so I'd expect this to be `kind:ps`

File: integration-test/2022-disputed-borders.py [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+                'kind': 'region',
+                'kind:US': 'region',
```
Comment (nvkelso) [inline integration-test/2022-disputed-borders.py:64]: Ditto: `kind:us`

File: integration-test/2022-disputed-borders.py [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+            })
+
+        # make sure kind:VN didn't make it in because its admin_level doesn't map to anything
+        self.assert_no_matching_feature(z, x, y, 'boundaries', {
+            'id': 726514231,
+            'kind:VN': None
```
Comment (nvkelso) [inline integration-test/2022-disputed-borders.py:70]: `kind:vn`

File: vectordatasource/transform.py [hunk 5]
```diff
@@ -9624,3 +9624,25 @@ def capital_alternate_viewpoint(shape, props, fid, zoom):
...
+def admin_level_alternate_viewpoint(shape, props, fid, zoom):
+    """
+    turns e.g. admin_level:XX=4 into kind:XX=region
+    """
+    admin_viewpoint_prefix = 'admin_level:'
+    tags = props['tags']
+
+    for k in tags.keys():
+        if k.startswith(admin_viewpoint_prefix):
+            viewpoint = k[len(admin_viewpoint_prefix):].upper()
```
Comment (nvkelso) [inline vectordatasource/transform.py:9641]: This should instead be `lower()` as Tilezen's convention is all lowercase property names (so `kind:us` not `kind:US`).

File: vectordatasource/transform.py [hunk 6]
```diff
@@ -9624,3 +9624,25 @@ def capital_alternate_viewpoint(shape, props, fid, zoom):
...
 
     return shape, props, fid
+
+
+_ADMIN_LEVEL_TO_KIND = {'2': 'country', '4': 'region', '6': 'county', '8': 'locality'}
+
+
+def admin_level_alternate_viewpoint(shape, props, fid, zoom):
+    """
+    turns e.g. admin_level:XX=4 into kind:XX=region
```
Comment (nvkelso) [inline vectordatasource/transform.py:9634]: For consistency, `kind:xx`

File: yaml/boundaries.yaml [hunk 7]
```diff
@@ -153,6 +153,25 @@ filters:
       claimed_by: {col: claimed_by}
       recognized_by: {col: recognized_by}
     table: osm
+  # these filters look for claims that don't have a claimed_by
```
Comment (nvkelso) [inline yaml/boundaries.yaml:156]: Expand the comment to explain in 2nd line:

> # but have different point of view tags for admin_level:XX

Comment (nvkelso) [reply yaml/boundaries.yaml:156]: Up above this section we have inline documentation like:

``` 
  # we track two kinds of features from OSM for indicating disputes:
  #
  # 1. **claims**, which have `boundary=claim, claimed_by=*, admin_level=*`
  #    tags. these indicate borders which aren't generally recognized, but
  #    should appear in the viewpoint of the `claimed_by=` value. additionally,
  #    extra viewpoints which don't make a claim for themselves but still want
  #    the claimant's view reflected in their own can be specified in a
  #    `recognized_by=*` tag.
```

Might add another sentence to the 1. point at the end about this new sub case?

**Review by nvkelso — APPROVED** [review]
Some nits, but otherwise LGTM

**Comment by travisgrigsby** [comment]
Cool, thanks so much for the comments :D.  Addressed everything.

**Comment by travisgrigsby** [comment]
Added 
```
#  There are also cases where boundary=claim but there
#    is no corresponding claimed_by - these will have admin_level:XX tags instead
#    that will dictate what kind each viewpoint should have.
```

**Pull request merged**